scripts/rhel_validation/check_lis_modules.sh

```diff
@@ -31,15 +31,15 @@ main() {
         printf "\nLoaded LIS modules after modprobe:\n"
     fi
     for module in ${UNSET_MOD[@]};do
-        dmesg --clear
+        dmesg -c 1> /dev/null 2>&1
         mod_out=$(modprobe $module)
         mod_exit=$?
         mod_log="$(dmesg)"
         lsmod | grep "${module} " > /dev/null
         mod_found=$?
         if [[ $mod_exit -eq 0 ]] && [[ $mod_found -eq 0 ]];then
             mod_ver="$(modinfo $module | grep -w version)"
-            mod_ver=${mod_ver#*:}
+	    [[ -z $mod_ver ]] && mod_ver="in-box driver" || mod_ver=${mod_ver#*:}
```

Comment:
from where does this "in-box driver" comes from?
For built-in drivers, the version is shown as 3.1 or matching the kernel.

Reply:
Our focus is on displaying LIS driver. So in-box driver is shown to make the reader aware that LIS driver was not loaded due to some reason and in-box driver is loaded.

Reply:
But where are you setting mod_ver to "in-box driver"?
if still needed be, can you please change to "built-in drivers" instead? That aligns with the other naming used like on technet - https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/supported-centos-and-red-hat-enterprise-linux-virtual-machines-on-hyper-v

Reply:
When the condition is true then it set the in-box driver. Now as per you comment, I have changed it to display "built-in driver"